.github/workflows/static-analysis.yml

```diff
@@ -73,9 +73,8 @@ jobs:
           key: turbo-${{ github.job }}-${{ github.ref_name }}-${{ github.sha }}
           restore-keys: turbo-${{ github.job }}-${{ github.ref_name }}-
 
-      # it is a local dep, as long as not published, need to build it before installing the rest or `pnpm i` will fail
-      - name: Build merkle-tree-wasm deps
-        run: pnpm --filter @anonklub/merkle-tree compile
+      - name: Build Rust wasm deps
+        run: pnpm --filter @anonklub/merkle-tree compile && pnpm --filter @anonklub/spartan-ecdsa compile
```

Comment:
Same as for test workflow

I think the `devPreinstall` hook makes this step now redundant.
```suggestion
```

Reply:
You were right.
After testing we need to manually run this otherwise the cache is hit. Because it only cares about node deps (pnpm lock file) and knows nothing about the changes that impact rust files...